staging/src/k8s.io/api/policy/v1beta1/types.go

```diff
@@ -84,6 +84,8 @@ type PodDisruptionBudgetStatus struct {
 // +k8s:prerelease-lifecycle-gen:introduced=1.5
 // +k8s:prerelease-lifecycle-gen:deprecated=1.22
```

Comment:
Should we announce deprecation in 1.21 since policy/v1. PodDisruptionBudget would be available?